HW_8/views.py

```diff
@@ -0,0 +1,53 @@
+from logger import log
+
+
+@log
+def greetings():
+    print('Список персонала')
+
+
+@log
+def show_menu():
+    while True:
+        try:
+            value = int(input(
+                '''\nВыберите команду:\n1 - Просмотреть весь список сотрудников.\n2 - Добавить сотрудника.\n3 - Изменить данные сотрудника.\n4 - Удалить сотрудника.\n0 - Выход.\n'''))
```

Comment:
Можно сделать так, чтобы смотрелось в коде понятнее.
```suggestion
                '\nВыберите команду:\n'
                '1 - Просмотреть весь список сотрудников.\n'
                '2 - Добавить сотрудника.\n'
                '3 - Изменить данные сотрудника.\n'
                '4 - Удалить сотрудника.\n'
                '0 - Выход.\n'))
```